src/wireframes/shapes/neutral/grid.ts

```diff
@@ -3,170 +3,207 @@
  *
  * @license
  * Copyright (c) Sebastian Stehle. All rights reserved.
-*/
-
-import { ConfigurableFactory, DefaultAppearance, Rect2, RenderContext, Shape, ShapePlugin } from '@app/wireframes/interface';
-import { CommonTheme } from './_theme';
-
-const HEADER_BACKGROUND_COLOR = 'HEADER_BACKGROUND_COLOR';
-const HEADER_FOREGROUND_COLOR = 'HEADER_FOREGROUND_COLOR';
-const HEADER_HIDDEN = 'HEADER_HIDDEN';
+ */
+
+import {
```

Comment:
I usually don't do that. I hate it that you make one additional import and then everything is formatted totally different.